website/templates/project/registration_editor_templates.mako

```diff
@@ -154,4 +154,28 @@
     </div>
 </script>
 
+<script>
+ function setAllBoxes(value) {
+     var boxes = document.querySelectorAll('input[type="checkbox"]');
```

Comment:
See previous note about selector- will this pick up checkboxes outside the modal as well?